hardware_interface/include/hardware_interface/actuator.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 ros2_control Development Team
+// Copyright 2021 ros2_control Development Team
```

Comment:
The copyright should always state the original date of authoring that file.
alternatively you could specify a range of the file, that is `2020 - 2021`. 

Same comment for the files below.

Reply:
```suggestion
// Copyright 2020 - 2021 ros2_control Development Team
```